docs/src/main/asciidoc/security-getting-started-tutorial.adoc

```diff
@@ -495,7 +502,8 @@ Dev Services for PostgreSQL supports testing while you develop by providing a se
 
 == Test your application in production mode by using Curl or browser
 
-To test your application using Curl or the browser, you must first start a PostgreSQL server, then compile and run your application either in JVM or native mode.
+To test your application using Curl, a browser, or the `quarkus dev` command, start a PostgreSQL server first.
```

Comment:
I think the`quarkus dev command` should not be used here. As this is testing in production mode. 

```suggestion
To test your application using Curl, or a browser, start a PostgreSQL server first.
```

Reply:
Thanks @jedla97, I applied the suggestion and then realized it was not my PR :-), sorry @rolfedh, but indeed, the suggestion is correct

Reply:
Updated. Thanks.

Reply:
Updated. Thanks.